packages/shared/src/hooks/usePlusSubscription.ts

```diff
@@ -15,11 +15,13 @@ export const usePlusSubscription = (): {
   showPlusSubscription: boolean;
   isPlus: boolean;
   isEnrolledNotPlus: boolean;
+  isOnboardingPlusActive: boolean;
   logSubscriptionEvent: (event: LogSubscriptionEvent) => void;
 } => {
   const { user } = useAuthContext();
   const isPlus = user?.isPlus || false;
   const plusSubscriptionFeature = useFeature(feature.plusSubscription);
+  const isOnboardingPlusActive = useFeature(feature.onboardingPlus);
```

Comment:
Oh the only issue here is that it will enroll everyone as soon as they hit onboarding page, and actually anytime this hook gets called even.

Did you check enrollment with ido in #experiment channel?

Reply:
Updated ✔️